pandas/io/gbq.py

```diff
@@ -353,10 +357,15 @@ def _parse_data(schema, rows):
 
     fields = schema['fields']
     col_types = [field['type'] for field in fields]
-    col_names = [field['name'].encode('ascii', 'ignore') for field in fields]
+    if compat.PY2:
+        col_names = [field['name'].encode('ascii', 'ignore') for field in fields]
```

Comment:
`utf-8 ` is ok. I think the conversion to ascii was a workaround for an unrelated issue numpy/numpy/2407 . 

For python 3 compatability, we can just use `str( )` . I've committed a new version.